### DISCUSSION TITLE
启动不了

### DISCUSSION THREAD

**Discussion by Folozz** [qier222/YesPlayMusic#1183] [Q&A] [answered]
# 尽量每个 issue 只提一个 bug 或新功能

### 提新 issue 前请确认 👉

- 没人提过这个 issue（[这里看所有 issue](https://github.com/qier222/YesPlayMusic/issues)）
- 项目的 Todo 里没有与你 issue 相关的内容（[这里看 Todo](https://github.com/qier222/YesPlayMusic/projects/1)）

### 反馈 bug 需要的信息

- 用的是网页版还是客户端   客户端
- 浏览器名称或电脑操作系统   win11
![image](https://user-images.githubusercontent.com/78134574/148163717-fb0caeb1-f89d-418e-9b6b-398be79acfbe.png)

- 控制台 Console 页面的截图（按 F12 可打开控制台）

**Comment by Folozz** [comment]
用不了了

**Comment by pan93412** [comment, marked as answer]
試試看重開機你的 Windows 系統？

**Reply by Folozz** [reply]
对 可以了
